### DISCUSSION TITLE
Home Assistant: Integration Tesla is not found

### DISCUSSION THREAD

**Discussion by askpet** [teslamate-org/teslamate#2710] [Q&A]
This part in confuguration.yaml:

tesla:
  username: !secret tesla_username
  password: !secret tesla_password
  scan_interval: 3600

gives me "Integration not found: tesla' when validating.

Can someone point me in the right direction? What am I missing

Kind regards
Anders

**Comment by agustind2004** [comment]
I have the same error. Can anyone help us?

**Reply by askpet** [reply]
I don't have the error. My solution was to just omit that part from the yaml file. The integration holds its own login, so I am not sure what this is for anyway.

Everything works as expected despite the removal.

Anders
